Adding_Filters

```diff
@@ -0,0 +1,171 @@
+                    <select id="vehicleType">
+                        <option value="">All</option>
+                        <option value="car">Car</option>
+                    </select>
+                </div>
+                <div class="filter-group">
+                    <label for="priceRange">Price Range:</label>
+                    <input type="range" id="priceRange" min="0" max="500" value="250">
+                    <span id="priceValue">$250</span>
+                </div>
+                <div class="filter-group">
+                    <label for="location">Location:</label>
+                    <select id="location">
+                        <option value="">All Locations</option>
+                        <option value="New York">New York</option>
+                        <option value="Mumbai">Mumbai</option>
+                        <option value="Delhi">Delhi</option>
+                        <option value="Chennai">Chennai</option>
+                        <option value="Bengaluru">Bengaluru</option>
+                    </select>
+                </div>
+                <div class="filter-group">
+                    <label for="rentalDuration">Rental Duration (days):</label>
+                    <input type="number" id="rentalDuration" min="1" value="1">
+                </div>
+                <div class="filter-group">
+                    <label for="userRating">User Ratings:</label>
+                    <select id="userRating">
+                        <option value="">All</option>
+                        <option value="1">1 Star & Up</option>
+                        <option value="2">2 Stars & Up</option>
+                        <option value="3">3 Stars & Up</option>
+                        <option value="4">4 Stars & Up</option>
+                        <option value="5">5 Stars</option>
+                    </select>
+                </div>
+                <button type="button" id="filterButton">Filter</button>
+            </form>
+        </div>
+
+        <div class="results">
+            <h2>Available Vehicles</h2>
+            <div id="vehicleList"></div>
+        </div>
+    </div>
+    
+    <script src="script.js"></script>
+</body>
+</html>
+
+###css##
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f4f4f4;
+    margin: 0;
+    padding: 20px;
+}
+
+.container {
+    max-width: 800px;
+    margin: 0 auto;
+    background: #fff;
+    padding: 20px;
+    border-radius: 5px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+h1, h2 {
+    color: #333;
+}
+
+.filters, .results {
+    margin-top: 20px;
+}
+
+.filter-group {
+    margin-bottom: 15px;
+}
+
+.filter-group label {
+    display: block;
+    margin-bottom: 5px;
+}
+
+#vehicleList {
+    margin-top: 20px;
+}
+
+.vehicle-item {
+    background: #f9f9f9;
+    padding: 10px;
+    margin-bottom: 10px;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+}
+
+##js##
```

Comment:
create a new js file to make this filter code reusable and import it to the page which would need the filtering component